### DISCUSSION TITLE
Read configuration from a file

### DISCUSSION THREAD

**Discussion by cherio** [erebe/wstunnel#267] [Ideas]
Having an ability to pass a configuration file (with e.g. one command line option per line) could be great from multiple perspectives:
* usability - no need to store multiple scripts when using/testing various configurations
* security - configuration could stored on encrypted devices
* privacy - ps command line won't reveal to everyone the details of the tunnel

**Comment by erebe** [comment]
Hello,
For this one, I keep it in todo, but I dont plan to imlement it in the near feature, as I am not very motivated to refactor a lot of main.rs to allow this feature. At some point if i reach the end of it, or if someone wants to make a PR for it

**Reply by cherio** [reply]
Understood. Thanks for considering!
